dasharo-security/me-keyword.resource

```diff
@@ -0,0 +1,9 @@
+*** Settings ***
+Library     BuiltIn
+Library     Process
+
+
+*** Keywords ***
+Check If ME Is Disabled
```

Comment:
This file should go into `lib/`, see how other keywords are distributed

Reply:
Please add documentation section, explaining what this keyword does, what it returns, what it means

Reply:
Either add to some existing library, or create a new one, and include it in the keywords.robot